t/dummy-isotovideo.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
```

Comment:
I recently learned that there might be better "best practices", see https://github.com/os-autoinst/scripts/pull/23/files#diff-9a7d9a4e0880c9871c35c54cd1a5f487R3-R4 . It's not at all critical and definitely not entirely necessary here but also shouldn't harm to do it right from the beginning. I am curious what you and others think about it. Also feel free to review https://github.com/os-autoinst/scripts/pull/23